aws/transport/http/client.go

```diff
@@ -222,3 +215,60 @@ func shallowCopyStruct(src interface{}) interface{} {
 
 	return dstVal.Interface()
 }
+
+func wrapWithoutRedirect(c *http.Client) *http.Client {
```

Comment:
Should this be exported as a utility?

Reply:
Would prefer not to, unless there is a demand for it.